skiko/src/commonTest/kotlin/org/jetbrains/skia/TextBlobTest.kt

```diff
@@ -0,0 +1,228 @@
+package org.jetbrains.skia
+
+import org.jetbrains.skia.tests.assertCloseEnough
+import org.jetbrains.skia.tests.assertContentCloseEnough
+import org.jetbrains.skia.tests.makeFromResource
+import org.jetbrains.skiko.kotlinBackend
+import org.jetbrains.skiko.tests.runTest
+import kotlin.math.cos
+import kotlin.math.sin
+import kotlin.test.*
+
+class TextBlobTest {
+
+    private val inter36: suspend () -> Font = suspend {
+        Font(Typeface.makeFromResource("./fonts/Inter-Hinted-Regular.ttf"), 36f)
+    }
+
+    private val eps = 0.001f // only this value works for kotlin/js
```

Comment:
Shall we select epsilon based on target platform?

Reply:
applied your suggestion